### PR TITLE
add context provider for preferred languages

### PR DESCRIPTION
**Pull request by anthopark**
* add context provider for preferred language support

### CODE REVIEW

**Comment by anthopark** [comment]
[Context API for Language State](https://trello.com/c/utxY7BOT/6-context-api-for-language-state)

File: nextjs-client/src/contexts/languageContext.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext, useState } from "react";
+
+const supportLanguages = Object.freeze({
+  English: Symbol("en"),
```
Comment (emilyPhee) [inline nextjs-client/src/contexts/languageContext.js]: Why used Symbol here?

File: nextjs-client/src/contexts/languageContext.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  Korean: Symbol("kr"),
+});
+
+const LanguageContext = createContext();
+
+const LanguageProvider = ({ children }) => {
+  const [preferredLanguage, setPreferredLanguage] = useState(
```
Comment (emilyPhee) [inline nextjs-client/src/contexts/languageContext.js:11]: question: do children have access to LanguageProvider and update the preferred Language state?

**Review by emilyPhee — APPROVED** [review]

File: nextjs-client/src/contexts/languageContext.js (hunk 1 above)
Comment (anthopark) [reply nextjs-client/src/contexts/languageContext.js]: using Symbol force to use the `supportLanguages` we defined here. 
ex. in the future we'll do: 
```
preferredLanguage === supportLanguages.English
// OR
setPreferredLangauge(supportLanguages.Korean)
````

File: nextjs-client/src/contexts/languageContext.js (hunk 2 above)
Comment (anthopark) [reply nextjs-client/src/contexts/languageContext.js:11]: The children of `LanguageProvider` will have access to `preferredLanguages` and `setPreferredLangauges` by using the `useLanguages()` I defined in this file.

**Comment by anthopark** [comment]
@emilyPhee Thanks for the review!

**Pull request merged**